src/sql/magic_cmd.py

```diff
@@ -33,6 +34,18 @@ def error(self, message):
 class SqlCmdMagic(Magics, Configurable):
     """%sqlcmd magic"""
 
+    displaycon = True
+    autolimit = None
+    style = "DEFAULT"
+    short_errors = True
+    displaylimit = None
+    autopandas = False
+    column_local_vars = False
+    feedback = False
+    autocommit = False
+    autopolars = False
+
+    @needs_local_scope
```

Comment:
same, do we need this?

Reply:
I need the `local_ns` in order to use `run(conn, sql, config, user_namespace)`. If I remove this decorator I don't have it.

Reply:
same, if we move to a `run_raw` function, we won't need this anymore